code-challenges.test.js

```diff
@@ -11,41 +11,150 @@
 // Run the file with the following command: $ yarn jest
 
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
+  // PseudoCode:
+    // input: an array of strings
+    // output: a subset of the initial array, with the zero-indexed element removed, and the remaining elements in shuffled order.
+    
+    // process: use the .reverse() method along with .pop() to remove the element that was initially at index 0. We then have to create a function that randomizes the array, and in this case we will use the Fisher-Yates method, which is a for-loop that iterates through the array from the end to the start. It swaps the the element at index "i" with the element at index "j" which is generated wiht the math.floor and math.random. Then using a temporary variable, we assign it the value at index "i", then overwrite that value with the element at index "j"
+    
+    // Process 2: I created another function with the help of a mentor, becuase I couldn't properly understand the Fisher-Yates method and spent the majority of class time on friday trying to understand it to no avail. With the instructors help, i realized i could use the slice() method to get rid of the first index without having to dedicate an entire line of code to it. Then i used .sort() with a function as a parameter to randomize the resulting array. We subtract math.random from .5 to generate an either positive or negative number, effectively randomizing our array.
```

Comment:
Great research! Either approach will definitely work for this challenge! The Fisher Yates is a little more complex but the main concept is the same